src/index.js

```diff
@@ -11,17 +11,71 @@ const nodePathParts = [
 process.env.NODE_PATH = nodePathParts.join(nodePathDelim)
 
 const core = require('@actions/core')
+const fetch = require('node-fetch').default
 const childProcess = require('child_process')
 const lhciCliPath = require.resolve('@lhci/cli/src/cli.js')
 const input = require('./input.js')
 const output = require('./output.js')
 
 // audit urls with Lighthouse CI
 async function main() {
+  let status
   core.startGroup('Action config')
   console.log('Input args:', input)
   core.endGroup() // Action config
 
+  /*******************************WARMING UP***********************************/
+  if (input.netlifySite) {
+    core.startGroup('Warming up')
+    const retryNumber = 5
+    let runs = 0
+    const sleep = async () => {
+      return new Promise(r => setTimeout(r, 60000))
```

Comment:
Maybe a good idea to pull out the `60000` ms out so that it can be modified easier/used to dynamically output how long the system is waiting in console logs.

Then maybe modify the logs to say `Waiting additional ${seconds} seconds for Netlify build to be done.`...

Reply:
I've been thinking about it for the next iteration after getting feedback, I didn't wanna overwhelm action settings :)